### DISCUSSION TITLE
Negative and Positive number

### DISCUSSION THREAD

**Discussion by sfmendes91** [beholdr/maska#175] [Q&A]
Is there a way of using maska for both negative and positive number?

For example:
**-99.10**
or
**99.10**
should be a valid value.

Unfortunately I didn't found anything in the documentation.

**Comment by vip9** [comment]
Have you found a way?
For me, the: 
```
            data-maska="m0.99"
            data-maska-tokens="m:-:optional|0:\d:multiple|9:\d:optional" 

```
works, but is there a better way?
